dvc/stage/__init__.py

```diff
@@ -36,16 +49,24 @@
 
 if TYPE_CHECKING:
     from dvc.dvcfile import DVCFile
+    from dvc.output import Output
+    from dvc.repo import Repo
+    from dvc.types import StrPath
     from dvc_data.hashfile.hash_info import HashInfo
     from dvc_objects.db.base import ObjectDB
 
 logger = logging.getLogger(__name__)
 # Disallow all punctuation characters except hyphen and underscore
 INVALID_STAGENAME_CHARS = set(string.punctuation) - {"_", "-"}
 Env = Dict[str, str]
+ChangedEntries = Tuple[List[str], List[str], Optional[str]]
```

Comment:
Let's make a namedtuple, or copy/paste it. I prefer less typealiases as possible.

Reply:
How about doing that in another PR, along with the change `status` in `dvc/stage/__init__.py` you suggested below?

```
def status(
        self, check_updates: bool = False, filter_info: bool = None
    ) -> Dict[str, List[Union[str, Dict[str, str]]]]:
```

Reply:
I'm fine with that. :)